pkgs/kde/misc/kommit/default.nix

```diff
@@ -0,0 +1,21 @@
+{
+  lib,
+  mkKdeDerivation,
+  fetchurl,
+  pkg-config,
+  libgit2,
+}:
+mkKdeDerivation rec {
+  pname = "kommit";
+  version = "1.6.0";
+
+  src = fetchurl {
```

Comment:
Do we have it already in the auto generated sources?

Reply:
No, it's not part of any of the large releases.